lib/inferno/entities/test_kit.rb

```diff
@@ -0,0 +1,133 @@
+require_relative '../repositories/test_kits'
+require_relative '../repositories/test_suites'
+
+module Inferno
+  module Entities
+    # @example
+    #
+    # module USCoreTestKit
+    #   class TestKit < Inferno::Entities::TestKit
+    #     id :us_core
+    #     title 'US Core Test Kit'
+    #     description <<~DESCRIPTION
+    #       This is a big markdown description of the test kit.
+    #     DESCRIPTION
+    #     suite_ids ['us_core_v311', 'us_core_v400', 'us_core_v501', 'us_core_v610']
```

Comment:
can this be inferred based on the TestSuites defined in the same module?

A somewhat open question i think is for TestKits that are related to other test kits and it may be useful to provide direct testing for those test kits in this test kit, how do we handle that?  Right now, by default we show the suites for imported test kits on the suite list page as a first-class suite, which is definitely imperfect but gives you more testing options.  We are in a new paradigm here, and i'm not sure how we want to handle it.  Should we just say 'nope, those are in different test kits, if you want to test those suites directly load up that test kit'.  Or something more nuanced like, having the ones loaded in the same namespace be 'primary' and have 'secondary/external/additional' ones that are either explicitly listed like this?

I think we probably should just come up with an answer this week and see how it flies, instead of leaving it open-ended indefinitely.

Reply:
I think we could do that. It would require looking through all of the constants in the test kit module namespace for test suites, and then recursively looking through any submodules for test suites. In US Core, for example, the suites are all in submodules based on the US Core version. This assumes that all suites belong to a top-level module (or a submodule at some level of nesting) which the test kit will also be a member of. I think that is a fair requirement which all of our test kits probably meet, but I don't know what's out in the wild.

I don't know that it's worth it if there is a possibility that we would then have to add more methods to include/exclude particular suites, though.